.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Continuous Integration'
+on: [ push, pull_request ]
+
+env:
+  OSSRH_USERNAME: ${{ secrets.SONATYPE_S01_USERNAME }}
+  OSSRH_TOKEN: ${{ secrets.SONATYPE_S01_PASSWORD }}
+  MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_CENTRAL_GPG_PASSWORD }}
+jobs:
+  BuildAndPublish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v3
+        with: # running setup-java again overwrites the settings.xml
+          java-version: 8
+          distribution: 'temurin'
+          cache: 'maven'
+          server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
+          server-username: OSSRH_USERNAME # env variable for username in deploy
+          server-password: OSSRH_TOKEN # env variable for token in deploy
+          gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
+      - name: Publish alfresco-value-assistance-repo packages
+        if: ${{ startsWith(github.ref, 'refs/heads/master') }}
```

Comment:
Maybe a more general question (to be checked with the team)
If it's possible to use "branches" keyword instead of the IF statement
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters